README.quickstart.md

```diff
@@ -1,29 +1,27 @@
 Easy-RSA 3 Quickstart README
 ============================
 
-This is a quickstart guide to using Easy-RSA version 3. Detailed help on usage
-and specific commands by running easyrsa with the 'help' command. Additional
-documentation can be found in the doc/ directory.
+This is a quickstart guide to using Easy-RSA version 3. Detailed help on usage and specific commands can be found by running `./easyrsa -h`.
```

Comment:
Can you remove the use of the backticks here?  Double quotes or no quotes at all would be better.  Backticks have special meaning in scripts and I want to avoid any potential error in interpretation.